lib/src/main/java/com/auth0/android/lock/Configuration.java

```diff
@@ -322,6 +327,14 @@ private void parseLocalOptions(Options options) {
         this.privacyURL = options.getPrivacyURL() == null ? "https://auth0.com/privacy" : options.getPrivacyURL();
     }
 
+    @StyleRes
+    public int authStyleForConnection(String strategy, String connection) {
+        if (authStyles.containsKey(connection)) {
+            return authStyles.get(connection);
+        }
+        return AuthConfig.styleForStrategy(strategy);
```

Comment:
In this case there's no need to expose a public static method, just return -1 or something like that when there's no config.
So the defaults are handled in AuthConfig itself, when constructed without a StyleRes (-1). Then the static method is private and it doesn't even need to be static